### PR TITLE
2022 theme update

### PR DESCRIPTION
**Pull request by SathyaBhat**

### CODE REVIEW

File: page-home.php [hunk 1]
```diff
@@ -1,211 +1,158 @@
...
+  </button>
+</div> -->
+<div class="container marketing">
+  <!-- START THE FEATURETTES -->
+
+  <div class="row featurette align-items-center first-featurette">
+    <div class="col-md-7">
+      <h2 class="featurette-heading">What is BarCamp Bangalore? <span class="text-muted">#bcb2022</span></h2>
+      <p class="lead">BarCamp Bangalore is an unconference – an ad-hoc gathering born from the desire for people to share and learn in an open environment.</p>
+      <p class="lead">BarCamp Bangalore is an opportunity for the Bangalore community to come together every year to learn, network, and discuss great ideas in person.</p>
```
Comment (SathyaBhat) [inline page-home.php:59]:
```suggestion
      <p class="lead">BarCamp Bangalore is an opportunity for people to come together to learn, network, and discuss great ideas in person.</p>
```

File: page-home.php [hunk 2]
```diff
@@ -1,211 +1,158 @@
...
+      <p class="lead">We're coming back on <span id="page_home_hero_date">November 5, 2022, Saturday</span>, at the <a href="https://goo.gl/maps/m7ARhpVeEHAWxYYGA" target="_blank">RV University, Bangalore</a>.</p>
+      <div class="calendar-options">
+        <script src="https://cdn.jsdelivr.net/npm/add-to-calendar-button@1" async defer></script>
+        <div class="atcb" >
+            {
+                "name":"Barcamp Bangalore",
+                "description":"BarCamp Bangalore is an opportunity for the Bangalore community to come together every year to learn, network, and discuss great ideas in person.",
```
Comment (SathyaBhat) [inline page-home.php:66]:
```suggestion
                "description":"BarCamp Bangalore is an opportunity for people to come together to learn, network, and discuss great ideas in person.",
```

File: page-home.php [hunk 3]
```diff
@@ -1,211 +1,158 @@
...
+    <div class="col-md-5">
+    <img height="500px" src="<?php bloginfo('template_url');?>/images/hero-02.jpg" />
+    </div>
+  </div>
+
+  <hr class="featurette-divider">
+
+  <div class="row featurette">
+    <div class="col-md-7 order-md-2">
+      <h2 class="featurette-heading">What is an Unconference? <span class="text-muted">Come and see for yourself.</span></h2>
```
Comment (SathyaBhat) [inline page-home.php:100]:
```suggestion
      <h2 class="featurette-heading">What is an unconference? <span class="text-muted">Come and see for yourself.</span></h2>
```

File: page-home.php [hunk 4]
```diff
@@ -1,211 +1,158 @@
...
+    <img height="500px" src="<?php bloginfo('template_url');?>/images/hero-01.jpg" />
+    </div>
+  </div>
+
+  <hr class="featurette-divider">
+
+  <div class="row featurette">
+    <div class="col-md-7">
+      <h2 class="featurette-heading">Meet the most outside-the-box thinkers in the city. <span class="text-muted">No, really.</span></h2>
+      <p class="lead">Sessions include topics ranging from design, technology, creativity, to entrepreneurship and lifestyle.</p>
```
Comment (SathyaBhat) [inline page-home.php:113]:
```suggestion
      <p class="lead">Sessions include topics ranging from design, technology, creativity, entrepreneurship, lifestyle, astronomy, music, and much more!</p>
```

File: page-home.php [hunk 5]
```diff
@@ -1,211 +1,158 @@
...
+    </div>
+  </div><!-- /.row -->
+
+  <hr class="featurette-divider">
+
+  <div class="row">
+    <div class="col-lg-12 text-center mb-10">
+      <h1 class="display-3 pb-5">Sponsors</h1>
+    </div>
+    <div class="col-lg-4">
```
Comment (the100rabh) [inline page-home.php:137]: Please dont forget to add FOSS United

File: page-home.php [hunk 6]
```diff
@@ -1,211 +1,158 @@
 <?php get_header();?>
-<div id="page_home_container" class="d-flex align-items-center justify-content-center text-center">
-    <div id="page_home_wrapper" class="flex-column">
-        <div id="page_home_namaskara" class="row">
-            <div class="col-12">Namaskara</div>
+<!-- <div id="myCarousel" class="carousel slide" data-ride="carousel">
```
Comment (the100rabh) [inline page-home.php:2]: I generally hate commented code being kept. Can we remove it

File: single.php [hunk 7]
```diff
@@ -2,88 +2,75 @@
...
-                    {
-
-                        $("#single_page_attendees_count").html(data.attendees_count + " Attending");
-                        $("#single_page_attendees_list").html(data.attendees_list);
-                        card.parent().html(data.button_text);
+                    else {
+                        // $("#single_page_attendees_count").html(data.attendees_count + " Attending");
+                        // $("#single_page_attendees_list").html(data.attendees_list);
+                        // card.parent().html(data.button_text);
+                        window.location.reload();
```
Comment (the100rabh) [inline single.php:25]: See if we can avoid this reload

**Pull request merged**